db/schema.rb

```diff
@@ -52,9 +53,12 @@
   end
 
   create_table "users", force: :cascade do |t|
-    t.string "name"
-    t.string "email"
+    t.string "email", null: false
     t.string "password_digest"
+    t.string "role", default: "user", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["email"], name: "index_users_on_email"
```

Comment:
I think this change is unrelated to the branch, maybe it is from a draft of an authentication strategy